File: .NET/Demo_Common/Enum/ColorType.cs
namespace Demo_Common.Enum
{
    [Flags]
    public enum ColorType
    {
        Black = 1,
        Red = 2,
        Yellow = 4,

        BlackRed = Black | Red,
        BlackYellow = Black | Yellow,
        BlackRedYellow = Black | Red | Yellow,
    }
}

File: Python/Enums/colorType.py
from enum import IntEnum


class ColorType(IntEnum):
    """
    Tag color type
    """
    Black = 1
    Red = 2
    Yellow = 4

    BlackRed = Black | Red
    BlackYellow = Black | Yellow
    BlackRedYellow = Black | Red | Yellow

    Color4 = 8
    Color6 = 9
    Color7 = 10
